core/shared/src/test/scala/scalaz/zio/internal/ExecutorSpec.scala

```diff
@@ -12,6 +12,7 @@ final class TestExecutor(val role: Executor.Role, val submitResult: Boolean) ext
   def shutdown(): Unit                    = ()
   def submit(runnable: Runnable): Boolean = submitResult
   def yieldOpCount: Int                   = 1
+  def metrics: None.type                  = None
```

Comment:
Why `None.type` vs `Option[ExecutionMetrics]`?

Reply:
I prefer it to be more precise (Inconsequential in test code, though)